cmd/drone-server/flags.go

```diff
@@ -95,12 +101,6 @@ var flags = []cli.Flag{
 		Usage:  "file path for the drone config",
 		Value:  ".drone.yml",
 	},
-	cli.StringFlag{
```

Comment:
I think we can promote documentation url somewhere in UI?

Reply:
I removed the flag as I do not see a reason an administrator would need to change that url, but it is visible in the UI. The "new" place I defined which url the docs are located at is: `server/web/config.go`.

Reply:
our use case is linking out to company internal docs around woodpecker and ci, rather than the public ones